### DISCUSSION TITLE
Which itwinui version should selected

### DISCUSSION THREAD

**Discussion by zhouDongW** [iTwin/iTwinUI#1079] [Q&A]
When I used @presentation-components 3.3.5, I found the error 'SassError Function hsl is missing argument $saturation' in the error scss file, I found that ~@itwin/itwinui-css/scss/style/theme was introduced. I tried @itwinui-css 0.63&1.0 and so on, but still got an error. And /scss path was not found in node_modules. Please tell me which version dependency I should choose here to solve this error

**Comment by mayank99** [comment]
Hey @zhouDongW, it's hard to tell why the error is happening without seeing the code or the full stacktrace, but if I had to guess, it's because you are using an old version of Sass or trying to pass a CSS file to your sass-loeader. Could you provide more details?

Also, I think you should open an issue in the [`presentation`](https://github.com/iTwin/presentation) repo, because `~@itwin/itwinui-css/scss/style/theme` is an internal path that they should *not* consume. It was removed in itwinui-css v1, and itwinui-css v0 is not maintained and will likely cause conflicts with v1, so really they need to upgrade to the latest version.
